cli/command/service/opts.go

```diff
@@ -529,6 +529,7 @@ type serviceOptions struct {
 	capAdd          opts.ListOpts
 	capDrop         opts.ListOpts
 	ulimits         opts.UlimitOpt
+	oomScoreAdj     opts.OomScoreAdj
```

Comment:
```suggestion
	oomScoreAdj     Uint64Opt
```

Reply:
That's an unisgned 64 bit integer for a value that ranges from -1000 to 1000

Reply:
My bad. You're correct; `Uint64Opt` would not be appropriate. No custom `Value` type is needed at all, since it's [just a signed int.](https://pkg.go.dev/github.com/spf13/pflag#FlagSet.IntVar)
```suggestion
	oomScoreAdj     int
```

Reply:
It was a custom type originally because I had a range constraint in place.  It should be fine, I need to fix the pipeline issues.

Reply:
> It was a custom type originally because I had a range constraint in place.

The history of how this PR has evolved is irrelevant. The custom type is not needed in the current iteration, therefore merging this PR with the custom type in place increases the ongoing maintenance cost of this project without any offsetting benefit.

> It should be fine, I need to fix the pipeline issues.

Please stay on topic.